src/index-metadbs/nmd/v1/links.json

```
@@ -0,0 +1,34 @@
+{
+   "meta" : {
+       "api_version" : "1.0.0",
+       "query" : {
+           "representation" : "/links"
+       },
+       "more_data_available" : false,
+       "schema" : "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+   },
+   "data" : [
+      {
+         "id" : "nmd",
+         "type" : "links",
+         "attributes" : {
+            "name": "novel materials discovery (NOMAD)",
+            "description": "A FAIR data sharing platform for materials science data",
+            "base_url": "http://nomad-lab.eu/prod/rae/optimade/",
+            "homepage": "http://nomad-lab.eu",
```

Would you prefer to use https links as you do in the providers.json?

---

For your convenience...

```suggestion
            "base_url": "https://nomad-lab.eu/prod/rae/optimade/",
            "homepage": "https://nomad-lab.eu",
```

---

Thx a lot.